__test__/api/organization-post.test.js

```diff
@@ -0,0 +1,25 @@
+const app = require("../../server");
+const supertest = require("supertest");
+const request = supertest(app);
+const { describeIfEndpoint } = require("../helpers/conditionalTests");
+
+describeIfEndpoint("POST", "/v1/organizations", "POST /v1/organizations", () => {
```

Comment:
replace `describeIfEndpoint(string, string, string)` with `describe(string)`
this is no longer required since all endpoints have been implemented

Reply:
Okay